### PR TITLE
Update create-secure-guest-sharing-environment.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about licensing requirement to use conditional access

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2169

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/solutions/create-secure-guest-sharing-environment.md [hunk 1]
```diff
@@ -97,6 +97,9 @@ To create a conditional access policy
 
 Now, the first time a guest user attempts to access content or a team or site in your organization, they will be required to accept the terms of use.
 
+> [!NOTE]
+> Using Conditional requires an Azure AD Premium P1 license. For more information, Please see [What is Conditional Access}(https://docs.microsoft.com/azure/active-directory/conditional-access/overview)
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/create-secure-guest-sharing-environment.md:101]:
```suggestion
> Using Conditional requires an Azure AD Premium P1 license. For more information, Please see [What is Conditional Access}(https://docs.microsoft.com/azure/active-directory/conditional-access/overview)
```
```suggestion
> Using Conditional Access requires an Azure AD Premium P1 license. For more information, please see [What is Conditional Access](https://docs.microsoft.com/azure/active-directory/conditional-access/overview).
```

Comment (JohanFreelancer9) [inline microsoft-365/solutions/create-secure-guest-sharing-environment.md]:
```suggestion
> Using Conditional Access requires an Azure AD Premium P1 license. For more information, please see [What is Conditional Access](https://docs.microsoft.com/azure/active-directory/conditional-access/overview).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**